ocaml-lsp-server/docs/ocamllsp/documentation-spec.md

```diff
@@ -0,0 +1,40 @@
+# Documentation Request
+
+## Description
+
+Merlin has a command `document` that gets `odoc` documentation for symbols based on a cursor position or an optional identifier. This request allows documentation to be gotten using a classic Merlin workflow.
+
+## Client capability
+
+```js
+export interface GetDocClientCapabilities {
+	contentFormat: MarkupKind[];
+}
+```
+- `contentFormat`: Client supports the following content formats if the content property refers to a `literal of type MarkupContent`. The order describes the preferred format of the client.
```

Comment:
We should probably add the capability relative to the request: `handleDocumentation`